ppl/zqd/db/postgresdb/ztests/intake.yaml

```
@@ -0,0 +1,85 @@
+tag: services
+
+script: |
+  # Start an authentication enabled zqd, and create 2 separate access tokens
+  # representing different users in different tenants.
+  ZQD_EXTRA_FLAGS="-auth.enabled=true -auth.clientid=testclient -auth.domain=https://testdomain -auth.jwkspath=./auth-public-jwks.json" source startup.sh
+  zapi -h $ZQD_HOST -configpath ./user1 auth store -access \
+    $(gentoken -domain https://testdomain -privatekeyfile ./auth-private-key -keyid testkey -tenantid tenant1 -userid user1)
+  zapi -h $ZQD_HOST -configpath ./user2 auth store -access \
+    $(gentoken -domain https://testdomain -privatekeyfile ./auth-private-key -keyid testkey -tenantid tenant2 -userid user2)
+  zapi -configpath ./user1 -h $ZQD_HOST auth verify
+  zapi -configpath ./user2 -h $ZQD_HOST auth verify
+  zapi_user1="zapi -configpath ./user1 -h $ZQD_HOST"
```

One man's opinion, but I think it would be better to only do to this:
```suggestion
  user1="-configpath ./user1 -h $ZQD_HOST"
```

And run zapi commands as:
```
zapi $user1 intake
```

I was disoriented when I looked at this file not seeing a the zapi command anywhere.

---

good idea, done in 453c71d936e10208d71345ce5747d67c09c0b921